### ISSUE TITLE
Try on Wiki data

### ISSUE THREAD

**Issue by meiliyang** [OptimalScale/LMFlow#364]
Hi, I tried to use the wiki data "wiki_en_eval_1k.json" and run "run_finetune.sh" with the default settings. When I run the chat bot and asking questions from the wiki content, it doesn't reply as in the wiki file. For this use case, which model should I use? and what parameters I need to try? Thank you

**Comment by research4pan** [comment]
Thanks for your interest in LMFlow! According to our experience, a 3b model can learn to produce acceptable responses, and 7b model can learn pretty well. If a model doesn't reply according to the training data, it is highly possible to be caused by underfitting, you may consider training it with more epochs and tune hyperparameters to reduce its training loss, including learning rates, schedulers, etc..

As a starting point, a llama-7b model will be good. If not available, gpt series model (gpt-neo, gpt-j, stablelm) with smaller sizes can be another option. Hope that answers your question. Thanks 😄

**Comment by shizhediao** [comment]
This issue has been marked as stale because it has not had recent activity. If you think this still needs to be addressed please feel free to reopen this issue. Thanks
